test/config-next/ocsp-updater.json

```diff
@@ -2,11 +2,9 @@
   "ocspUpdater": {
     "dbConnectFile": "test/secrets/ocsp_updater_dburl",
     "maxDBConns": 10,
-    "newCertificateWindow": "1s",
```

Comment:
I think this should be done in `test/config` as well. Ditto for the `newCertificateBatchSize` diff in this file.